src/client/components/equalizer/equalizer.js

```diff
@@ -255,8 +255,10 @@ class Equalizer extends Component {
       <Button
         className="ButtonStyleTemplate"
         onClick={pauseSoundFromFile}
-        icon={pauseIcon}
-        value="Pause"
+        icon={this.props.audioData.loading ? null : pauseIcon}
```

Comment:
U can add "loading" in destruction (218 line).

Reply:
done